src/Routers/routes/FacilityRoutes.tsx

```diff
@@ -14,6 +15,9 @@ import FacilityLocationRoutes from "@/Routers/routes/FacilityLocationRoutes";
 const FacilityRoutes: AppRoutes = {
   "/facility": () => <FacilityList />,
   "/facility/create": () => <FacilityCreate />,
+  "/facility/:facilityId/cover": ({ facilityId }) => (
```

Comment:
`/cover` is not very meaningful. want to cover the facility?

```suggestion
  "/facility/:facilityId/upload-cover-image": ({ facilityId }) => (
```